Commands/Anime/kiss.js

```diff
@@ -30,26 +30,10 @@ module.exports = {
             );
         } else {
             const { url } = await kiss().catch(() => { });
-
             if (!url) return message.channel.send(`Could not connect to nekos.life`);
-
-            if (
-                message.mentions.members.size &&
-                message.mentions.members.first().id === message.author.id
-            ) {
-                return message.channel.send(`S~seriously?!`);
-            } else if (message.mentions.members.size) {
-                return message.channel.send(
-                    embed
-                        .setColor(config.embedcolor)
-                        .setDescription( `${message.member} kisses ${message.mentions.members.first()}!`)
-                        .setImage(url)
-                );
-            } else {
-                return message.channel.send(
-                    `Sorry ${message.member}, I can't seem to locate your imaginary friend.`
-                );
-            }
+            if (message.mentions.members.size && message.mentions.members.first().id === message.author.id) return message.channel.send(`S~seriously?!`);
+            if (message.mentions.members.size) return message.channel.send({ embeds: [embed.setColor(config.embedcolor).setDescription( `${message.member} kisses ${message.mentions.members.first()}!`).setImage(url)]});
+            return message.channel.send(`Sorry ${message.member}, I can't seem to locate your imaginary friend.`);

```

Comment:
You should use 
```js
message.channel.send({content: ""})
```

Reply:
If it's string only, you can use message.channel.send("string")